risc0/zkp/src/prove/prover.rs

```diff
@@ -40,7 +40,6 @@ fn make_coeffs<H: Hal>(hal: &H, witness: &H::Buffer<H::Elem>, count: usize) -> H
     // Do interpolate
     hal.batch_interpolate_ntt(&coeffs, count);
     // Convert f(x) -> f(3x), which effective multiplies coefficients c_i by 3^i.
-    #[cfg(not(feature = "circuit_debug"))]
```

Comment:
This is actually useful for proof system development. In fact it'd be good to go further and make this feature flag work on other areas where we introduce entropy.

Reply:
Ok, I'll add it back and add it to the feature flags listed in `Cargo.toml`

Reply:
Added it back in ffb6c219fc121a7307687c159b419dd98da229e1